### DISCUSSION TITLE
Specify served Folderprefix within Bucket

### DISCUSSION THREAD

**Discussion by Dentling** [nginxinc/nginx-s3-gateway#154] [Q&A] [answered]
Hello,

I recently set up the s3-gateway and it works very well.
But I was not able to find a way to specify the served content via a "folder-prefix".

To explain:

I have my bucket and on that I have multiple Folders including one "public"-Folder which is this the only one I would like to serve via the nginx.

So the folder directory should start inside of that "public"-folder.

Is this possible somehow or is this not recommend because I should create a new bucket for the "public"-stuff.

Thanks in advance.

**Comment by dekobon** [comment, marked as answer]
I think for your use case, it would be better to have a separate bucket for the public folder. You could modify the scripts such that you can achieve what you want to do, but it will add a fair but of complexity to the gateway and make it less of a generalized solution.
